### PR TITLE
change

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: deep/deep-dashboard-tour.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# DEEP Dashboard tour
+
+<kbd>![sign-in](images/deep-dashboard/deep-dashboard-tour.png)</kbd>
+
+1 - Device Information
```
Comment (SenthilKumarSivakami) [inline deep/deep-dashboard-tour.md:5]: Instead of plain "1" can we use the image itself 
item 1

Comment (SenthilKumarSivakami) [reply deep/deep-dashboard-tour.md:5]: and then say,  Displays details of the device onboarded to SEED.

Comment (SenthilKumarSivakami) [reply deep/deep-dashboard-tour.md:5]: If ok, pls apply the same logic for this document.

**Review by SenthilKumarSivakami — CHANGES_REQUESTED** [review]

File: deep/deep-dashboard-tour.md
Comment (kpslp) [inline deep/deep-dashboard-tour.md:1]: hi @SenthilKumarSivakami changes made. do you think it looks okay without *-* in between the step image and the item?

**Pull request closed without merging**